Sources/Serialized.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Serialized.swift
+//  FTAPIKit-iOS
+//
+//  Created by Patrik Potoček on 27/06/2019.
+//  Copyright © 2019 FUNTASTY Digital s.r.o. All rights reserved.
+//
+
+import Foundation
+
+// This class provides user with easy way to serialize access to a property in multiplatform environment. This class is written with future PropertyWrapper feature of swift in mind.
+final class Serialized<Value> {
+    
+    // Synchronization queue for the property. Read or write to the property must be perforimed on this queue
```

Comment:
```suggestion
    /// Synchronization queue for the property. Read or write to the property must be perforimed on this queue
```

Reply:
This is private property. Are you sure that it needs public documentation?

Reply:
Sorry for the unclear formulation, I just want this documentation to be visible when using the class from other files in the framework.